src/main/java/org/libsdl/app/SDLOnTouchListener.kt

```diff
@@ -85,3 +86,13 @@ private fun MotionEvent.touchValues(i: Int): TouchValues {
             min(this.getPressure(i), 1.0f)
     )
 }
+
+class TouchParameters(
```

Comment:
nit: not sure if it makes any difference to use a [`data class` ](https://kotlinlang.org/docs/data-classes.html) here instead. would be slightly more kotlin-idiomatic since we're really just storing some data here.